regolith/builders/preslistbuilder.py

```diff
@@ -31,30 +31,81 @@ def construct_global_ctx(self):
         gtx['str'] = str
         gtx['zip'] = zip
 
+    def group_member_ids(self, grp):
+        """Get a list of all group member ids
+
+        Parameters
+        ----------
+        grp: string
+            The id of the group
+
+        Returns
+        -------
+        list:
+            The list of ids of the people in the group
+
+        Notes
+        -----
+        - Groups that are being tracked are listed in the groups collection
+        with a name and an id.
+        - People are in a group during an educational or employment period.
+        - To assign a person to a tracked group during one such period, add
+        a "group" key to that education/employment item with a value
+        that is the group name.
+        - This function takes the group id that is passed and searches
+        the people collection for all people that have been
+        assigned to that group in some period of time and returns a list of
+        """
+        grpmembers = []
+        for person in self.gtx['people']:
+            for position in person.get('education', {}):
```

Comment:
Would it be more concise to for loop through education and group the keys?
eg
```python
grpmembers = {}
for person in self.gtx['people']:
    for k in ['education', 'employment']:
        for position in person.get(k, {}):
            grpmembers.add(person['_id'])
```

Reply:
Thanks Chris, that is cleaner.